pkg/metrics/statsd/client.go

```diff
@@ -3,10 +3,11 @@ package statsd
 import (
 	"fmt"
 	"net/http"
-	"net/http/httputil"
+	stdhttputil "net/http/httputil"
```

Comment:
Should be this the other way around?
We keep the standard lib name and we rename ours?

Reply:
most of the time I saw aliasing the standard package with `std` prefix, but I don't mind either way.